tests/harness/video_utils.py

```
@@ -0,0 +1,130 @@
+    cmd = [
+        "ffprobe",
+        "-v",
+        "quiet",
+        "-print_format",
+        "json",
+        "-show_format",
+        "-show_streams",
+        "-count_frames",
+        "pipe:0",
+    ]
+    result = subprocess.run(cmd, input=video_bytes, capture_output=True, check=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)